Update JWT-JWKS-authentication-org-management.md

lemonez opened the pull request:
fix some grammar, improve formatting, update placeholder string names

no JIRA ticket just saw some issues. haven't updated screenshots. made a ticket to replace screenshots of code with actual example code.

## Checklist
- [ ] My PR title includes a relevant Jira ticket name
- [x] If no associated Jira ticket, my PR title is informative (individual commit messages are squashed in this repo)
- [x] Unnecessary text (Notes, TODOs, etc.) has been removed
- [ ] Screenshots, if present, are full-width and follow the [Screenshot guidelines](https://algorithmia.atlassian.net/wiki/spaces/CUSTOMERS/pages/1634861478/CFD+Style+Guide#Screenshots)
- [x] Grammar, style, tone, punctuation, etc. follow the Algorithmia [manual of style](https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing)
- [x] Headers are “Sentence case with Proper Nouns capitalized”
- [x] Placeholder strings are UPPER_SNAKE_CASE and are named as in the [Glossary](https://docs.google.com/document/d/1bYs0j_a4v8LbkbS8r0x2SZ_J6mZ4sIt4w_60dq8iGh4/edit#heading=h.qov2yks2a685) where appropriate

lemonez committed:
Update JWT-JWKS-authentication-org-management.md

fix some grammar, improve formatting, update placeholder string names

warmbowski reviewed (dismissed):
Looks like it's more clarity updates than content.  lgtm

lemonez committed:
Update JWT-JWKS-authentication-org-management.md

change `GROUP_NAME` to `ORG_NAME` to match placeholder naming scheme

lemonez committed:
Update JWT-JWKS-authentication-org-management.md

decided it was clearer to specify placeholder as `ALGO_OWNER` which is consistent with other pages
